### ISSUE TITLE
`break-inside: avoid` ignored when applied to grid element

### ISSUE THREAD

**Issue by krksgbr** [pagedjs/pagedjs#31]
Hello! :wave: 

I'm using css grid to layout my table of contents. It seems that `break-inside: avoid` does not work when applied to a grid item.

I found  the`avoidBreakInside` method in `chunker/layout.js` that seems to be relevant, however it does not appear to be called anywhere. I tried to figure out how to fix this issue, but I got lost. My idea was to check if `break-inside: avoid` is present on an element, or ancestor of an element causing the overflow, and if so, make sure the page break happens before the element.
I would appreciate any pointers on how to achieve this. Thank you!

**Comment by aammfe** [comment]
Yes same here its is very frustrating, even in flexbox its not working. `break-inside: avoid` don't work with it as well.
